src/app/parcels/ActionBar/DuplicateDownloadWarning.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import React from "react";
+import Alert from "@mui/material/Alert/Alert";
+import styled from "styled-components";
+import { displayPostcodeForHomelessClient } from "@/common/format";
+
+export interface DuplicateDownloadWarningProps {
+    postcodes: (string | null)[];
+}
+
+const StyledAlert = styled(Alert)`
+    border-radius: 4px;
+`;
+
+const DuplicateDownloadWarning: React.FC<DuplicateDownloadWarningProps> = (props) => {
+    return (
+        <StyledAlert severity="warning">
+            The following postcodes have already been downloaded:{" "}
+            {props.postcodes
+                .map((postcode) => postcode ?? displayPostcodeForHomelessClient)
```

Comment:
Should we just ignore the ones that don't have a postcode?

Reply:
I wasn't sure about this. Originally I did ignore it, but I merged this change from #326 and then thought that the postcodes shown here really just represent parcels which already have labels/lists printed. So if a parcel is going to have a duplicate label we should still flag that, even if it doesn't have a postcode. What do you think?

Reply:
So if the client is deleted and has a `-` as postcode I don't think we need to warn against that - the labels that come out are blank and shouldn't cause any issues (and hopefully it shouldn't really happen anyways). From what I imagine happens now is it shows NFA for both and we'd want to have something like 
```
if isActive then  
   if postcode then 
      postcode 
   else 
      displayPostcodeForHomelessClient 
else 
    don't add 
```
but that might be complicated

Not too sure about NFA - maybe it is good to point it out even if it is a bit confusing

So maybe leaving it as is is the best way to go about it